riscv-overlay-software-standard-draft.adoc

```diff
@@ -221,11 +219,11 @@ On RTOS based system, the the RT-Engine will provide hooks to protect its critic
 
 The RT-Engine should support a system bare metal design and/or RTOS system design.
 
-1.	The implementation with/without RTOS should be a build time options.
-2.	If RTOS is supported, the RT-Engine should be “thread save” and not blocking other threads due to overlay operations.
-3.	Blocking can be acceptable for short critical section and only with inherent operations (e.g. mutex)
-4.  The RT-Engine should be agnostics to any specific RTOS, therefore hooks should be provided _(link:#rtos-hooks[RTOS hooks])_
-5.	Load operation should lock the designated memory region in the heap, to prevent a case were higher priority task will take the region from the current running task.
+1.	The implementation with/without RTOS should be a build time option.
+2.	If RTOS is supported, the RT-Engine should be thread safe and not block other threads due to overlay operations.
```

Comment:
I think ```thread safe``` should be at leas ```thread-safe``` since some RTOSs calling it threads and some calling it ```tasks```
What do you think ?